custom/conf/app.example.ini

```diff
@@ -1075,8 +1079,8 @@ LEVEL = Info
 ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
 ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
 ;;
-;; Path for local repository copy. Defaults to `tmp/local-repo` (content gets deleted on gitea restart)
-;LOCAL_COPY_PATH = tmp/local-repo
+;; Path for local repository copy. Defaults to `local-repo` under `TEMP_PATH` except it's an absolute path (content gets deleted on gitea restart)
```

Comment:
`content gets deleted on gitea restart`: is it still true?

If it is true, suppose there are 2 processes:

1. `gitea dump`
2. `gitea web`

Then restarting `gitea web` will corrupt `gitea dump`?

Reply:
I removed the logic to delete the temporary directories when start Gitea b88c3a6af3 . Looks like the previous logic is a bug. When a user start Gitea second time wrongly. Even if Gitea exit, the temporary directories will be deleted even another Gitea process is running.

Reply:
No, I don't think it is a bug. There are different temp directories.

And it seems that `gitea web` should delete its own temp files. Otherwise, there will be more and more garbage files.

Reply:
At least, it should be able to delete out-dated temp files (for example: 30 days or so)

Reply:
> No, I don't think it is a bug. There are different temp directories.
> 
> And it seems that `gitea web` should delete its own temp files. Otherwise, there will be more and more garbage files.

All temporary files and directories should be removed when the process ends, except in cases where Gitea exits abnormally. I think a clean cron task is better than deleting them when started.